
function calling() {
    try {
        callHook();
    } catch (error) {

    }
}